ingest/sqs/pools/ingester/pool_ingester.go

```diff
@@ -322,7 +322,7 @@ func (pi *poolIngester) convertPool(
 	defer func() {
 		r := recover()
 		if r != nil {
-			telemetry.IncrCounter(1, "sqs_ingest", "convert_pool", "panic", strconv.FormatUint(pool.GetId(), 10))
+			telemetry.IncrCounter(1, "sqs_ingest_convert_pool_panic", "pool_"+strconv.FormatUint(pool.GetId(), 10))
```

Comment:
@alessandrolomanto 

3. Alert and no dashboard please 🙏 

Name should be `sqs_ingest_convert_pool_panic`

Reply:
What values can the counter take: 0 or 1? Should the alert be triggered when it differs from 0?

Reply:
If it increases, we should get alerted